front/src/modules/trainschedule/components/ManageTrainSchedule/hooks/isValidInitalSpeed.ts

```diff
@@ -0,0 +1,6 @@
+const isInvalidInitalSpeed = (speed: number): boolean => {
+  const speedString = speed.toString();
+  return speedString.includes('.') && speedString.split('.')[1].length > 1;
+};
```

Comment:
This util could be more generic and added in utils/number. We also have an util to check for float number already. If you could also add a doc for the function and a few tests (with no decimals, with 0, with regular decimal etc). 

```suggestion
const isInvalidFloatNumber = (value: number, numberOfDecimal: number): boolean => {
  if (!isFloat(value)) return false
  const stringifyValue = value.toString();
  return stringifyValue.split('.')[1].length > numberOfDecimal;
};
```

Reply:
You're right. It's done